app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+from app.players.player import Player
+from app.players.elves.elf import Elf
+from app.players.dwarves.dwarf import Dwarf
+
+
+def calculate_team_total_rating(players_list: list[Player]) -> int:
+    return sum(player_rate.get_rating() for player_rate in players_list)
```

Comment:
why do u call variable as player_rate, we will have a player instance inside this variable